lsp/src/saros/lsp/extensions/ISarosLanguageServer.java

```diff
@@ -0,0 +1,13 @@
+package saros.lsp.extensions;
+
+import org.eclipse.lsp4j.jsonrpc.services.JsonDelegate;
+import org.eclipse.lsp4j.services.LanguageServer;
+import saros.lsp.extensions.server.account.IAccountService;
+
+/** Interface of the Saros language server. */
```

Comment:
Please add an abstraction description what this server interface does in this context. Is this a step which is necessary for extending the protocol?

Reply:
yes